src/features/wallet/send/SendNavigator.tsx

```diff
@@ -1,15 +1,27 @@
 import * as React from 'react'
 import { createStackNavigator } from '@react-navigation/stack'
+import { RouteProp } from '@react-navigation/native'
 import SendScreen from './SendScreen'
 import ScanScreen from '../scan/ScanScreen'
 import SendCompleteScreen from './SendCompleteScreen'
+import { RootStackParamList } from '../../../navigation/main/tabTypes'
 
 const SendStack = createStackNavigator()
 
-const SendNavigator = () => {
+type Route = RouteProp<RootStackParamList, 'Send'>
+
+type Props = {
+  route: Route
+}
+const SendNavigator = ({ route }: Props) => {
+  const scanResult = route?.params?.scanResult
```

Comment:
I think this could be written like this to avoid the inline object

```ts
  const { scanResult } = route?.params || {}
  return (
    <SendStack.Navigator headerMode="none">
      <SendStack.Screen
        name="Send"
        component={SendScreen}
        initialParams={scanResult}
      />
```

Reply:
Udpated